worlds/tloz/ItemPool.py

```diff
@@ -2,6 +2,8 @@
 from .Locations import level_locations, all_level_locations, standard_level_locations, shop_locations
 from .Options import TriforceLocations, StartingPosition
 
+from collections import Counter
```

Comment:
Technically, this import should be at the top

Reply:
I don't see a mention of import order in the style guide. Is the desired order mentioned somewhere?

Reply:
https://peps.python.org/pep-0008/#imports

>Imports should be grouped in the following order:
> 1. Standard library imports.
> 1. Related third party imports.
> 1. Local application/library specific imports.
> 
> You should put a blank line between each group of imports.